crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -82,14 +82,42 @@ type ChangeEpochTransaction {
 }
 
 type Checkpoint {
+	"""
+	The digest of the checkpoint
+	"""
 	digest: String!
+	"""
+	Sequence number of the checkpoint
+	"""
 	sequenceNumber: Int!
+	"""
+	Timestamp of the checkpoint
+	"""
 	timestamp: DateTime
+	"""
+	The aggregate authority signature
+	"""
 	validatorSignature: Base64
+	"""
+	The digest of the previous checkpoint
+	"""
 	previousCheckpointDigest: String
+	"""
+	A single commitment of ECMHLiveObjectSetDigest
+	"""
 	liveObjectSetDigest: String
```

Comment:
We should avoid comments that just add English filler words around the existing variable names, etc. (e.g. "The digest of the checkpoint").

Instead here, can you define the field (you should be suspicious of a definition that includes the word you're defining) explaining how the field is calculated and/or what it's used for? For example:

- `sequenceNumber` worth explaining that checkpoints have a global ordering that everyone agrees on.
- `timestamp` what is this a timestamp of i.e. what happened to the checkpoint at this time? Is this the time that's decided by consensus, or is it the time that the checkpoint was indexed, or something in between?
- `validatorSignature`, what do these signatures prove?
- `previousCheckpointDigest` -- previous in what sense? (is it the one with the previous sequence number?)
- `liveObjectSetDigest` - What does this hash attest to?

Reply:
just waiting to hear back from some folks who have context into these clarifying qs

Reply:
lmk if the new comments look good